inst/tests/tests.Rraw

```diff
@@ -17263,3 +17263,10 @@ if (identical(x, enc2native(x))) {
 
 # fintersect now preserves order of first argument like intersect, #4716
 test(2163, fintersect(data.table(x=c("b", "c", "a")), data.table(x=c("a","c")))$x, c("c", "a"))
+
+
+# Test new feature %notin%, #4152
+test(2164.1, 11 %notin% 1:10, TRUE)
+test(2164.2, "a" %notin% c(), TRUE)
+test(2164.3, "a" %notin% c("a", "b", "c"), FALSE)
+test(2164.4, c(1, 2) %notin% c(1,2,3), c(FALSE, FALSE))
```

Comment:
could you add some edge case tests as well (`character()`, `integer()` and `NULL`)? Should be fine since we're building on `%in%` but just in case.